samples/RemoteAuth/Bearer/RemoteBearerCore/Program.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.AspNetCore.SystemWebAdapters;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddReverseProxy().LoadFromConfig(builder.Configuration.GetSection("ReverseProxy"));
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+
+builder.Services.AddSystemWebAdapters()
+    .AddRemoteAppAuthentication(true, options =>
```

Comment:
What is the `true` parameter?

Reply:
That's specifying whether remote app auth should be the default scheme or not. I updated the docs about that, but I'll add a comment here, as well.